UI/ChatGPT/ChatGPT/ChatGPT/Services/IChatService.cs

```diff
@@ -0,0 +1,9 @@
+using ChatGPT.Business;
+
+namespace ChatGPT.Services;
+public interface IChatService
+{
+	ValueTask<ChatResponse> AskAsync(string prompt);
```

Comment:
Please use cancellation token in samples for better quality of code.

```suggestion
	ValueTask<ChatResponse> AskAsync(string prompt, CancellationToken ct = default);
```

Reply:
Done in next commits.